### PR TITLE
remove vnc dependencies

### PR DESCRIPTION
**Pull request by jshihstsci**

### CODE REVIEW

File: deployments/jwebbinar/image/environments/post-start-hook [hunk 1]
```diff
@@ -8,4 +8,16 @@ set -eu
 # be performed both on AWS and standalone.  AWS-only functions should be included
 # directly in common.yaml or perhaps another script.
 
+rsync -r /etc/default-home-contents/ /home/jovyan
+
 /opt/environments/setup-notebooks
+
+chmod +x /home/jovyan/Desktop/*.desktop
```
Comment (cslocum) [inline deployments/jwebbinar/image/environments/post-start-hook:15]: I'm not sure that we need this.  What files fit the `*.desktop` pattern?

File: deployments/jwebbinar/config/common.yaml [hunk 2]
```diff
@@ -67,6 +59,10 @@ jupyterhub:
           #        - name: mirage-data-volume
           #mountPath: /home/shared/mirage-data
           #readOnly: true
+    lifecycleHooks:
+      postStart:
+        exec:
+          command: ["/opt/environments/post-start-hook"]
```
Comment (cslocum) [inline deployments/jwebbinar/config/common.yaml:65]: I assume this was verified as working?

**Review by cslocum — APPROVED** [review]
Looks great!  Approved.

I did leave a few comments, at least one line needs to be cleaned up.

File: deployments/jwebbinar/image/environments/post-start-hook (hunk 1 above)
Comment (jshihstsci) [reply deployments/jwebbinar/image/environments/post-start-hook:15]: ok, will delete this line

**Pull request merged**